src/constants/__test__/dataTypes.test.js

```diff
@@ -0,0 +1,54 @@
+import { dataTypes } from '..'
+
+describe('dataTypes', () => {
+	describe('constant buckets', () => {
+		it('should be objects', () => {
+			const bucketNames = Object.keys(dataTypes)
+			bucketNames.forEach((bucketName) => {
```

Comment:
This can be reduced to just `Object.keys(dataTypes).forEach()` without losing readability, IMO.

You can even go further and use `Object.entries(dataTypes).forEach(([bucketName, bucket]) => {…})`, but I find that to be a little illegible, at least until `Object.entries` behavior becomes more common knowledge.

This comment applies to basically this entire file, but I leave choosing to regard it up to your discretion.

Reply:
Totally hear you -- I went with the more verbose approach because {{ testing }} but admit I don't have my gauge finely tuned just yet.

The logic here was the [AAA](https://github.com/goldbergyoni/javascript-testing-best-practices#-%EF%B8%8F-12-structure-tests-by-the-aaa-pattern) pattern -- specifically the spirit of the "arrange" part.  And by giving it an explicit name we tell the developer "we are explicitly iterating over the concept of bucket names."

But I get you that this is kind of implicit by the fact that we have `bucketname`.

Reply:
Right, I think that's why I'm comfortable with it; it doesn't prevent you from naming the individual items, which is where the clarity comes in. In fact, by naming the array of keys just to kick off the iterator, I actually start to lose track of the important thing: I have to read extra-careful to tell if subsequent operations with `bucketName` are actually `bucketNames`.

Reply:
(Although I appreciate you linking to the specific testing guidance you were trying to adhere to. That is such a good guide. I still think this legibility can be improved by inlining this one, but, leave it to you.)

Reply:
Makes sense!  Fixed up.